projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss

```diff
@@ -17,27 +17,44 @@
     margin: 0 0 $space-16;
     transition: all 0.3s ease-in-out;
 
-    nb-user {
-      &::ng-deep {
-        .user-picture.image {
-          height: 3.5rem;
-          width: 3.5rem;
+    &:hover {
+      background: rgba(0, 0, 0, 0.05);
+    }
+
+    nb-card-body {
+      padding: 0;
+
+      .user-image {
+        min-height: 3.5rem;
+        min-width: 3.5rem;
+        max-height: 3.5rem;
+        max-width: 3.5rem;
```

Comment:
Is it possible that we change this rem to px?

Reply:
Done!